### PR TITLE
More CMake love

### PR DESCRIPTION
**Pull request by anselmolsm**
- Add install rule
- Move CMakeLists to project's root dir (In fact it was a split)

### CODE REVIEW

**Comment by anselmolsm** [comment]
Repeating my comment from #10: I am not that happy with INSTALL_PATH, suggestions are welcome.

**Comment by dakerfp** [comment]
LGTM. I think that GAZEBO_PATH is perfectly ok

**Comment by anselmolsm** [comment]
I think GAZEBO_PATH is confusing, as gazebo itself has GAZEBO_PLUGIN_PATH and GAZEBO_MODEL_PATH. Maybe GAZEBO_SITL_PATH, GZSITL_PATH...

**Comment by guiccbr** [comment]
What about GZSITL_INSTALL_PATH?

File: README.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ once with:
 3. Set environment variables and open the Gazebo gzsitl_drone_target world file with the following command:
 
     ```
-    GAZEBO_PLUGIN_PATH=${GAZEBO_PLUGIN_PATH}:${PWD}/gzsitl/build GAZEBO_MODEL_PATH=${GAZEBO_MODEL_PATH}:${PWD}/models gazebo ./world/gzsitl_drone_target.world --verbose
+    GAZEBO_PLUGIN_PATH=${GAZEBO_PLUGIN_PATH}:${PWD}/build GAZEBO_MODEL_PATH=${GAZEBO_MODEL_PATH}:${PWD}/models gazebo ./world/gzsitl_drone_target.world --verbose
```
Comment (guiccbr) [inline README.md:64]: The ful path to the .so has to be given, as follows:
GAZEBO_PLUGIN_PATH=${GAZEBO_PLUGIN_PATH}:${PWD}/build/**gzsitl**

**Comment by anselmolsm** [comment]
See #15

**Pull request closed without merging**